pipeline/06-upload.R

```diff
@@ -10,7 +10,9 @@ suppressPackageStartupMessages({
   library(arrow)
   library(aws.s3)
   library(aws.ec2metadata)
+  library(ccao)
   library(dplyr)
+  library(git2r)
```

Comment:
**[Question, non-blocking]** Where is `git2r` being used in the `upload` stage?

Reply:
It's being used in `finalize`, see above comment.